On OS X, I get a bunch of compiler errors when building using your instructions. They seem to be related to a lack of C++11 support:

```
../src/firmware.hpp:25:9: error: unknown type name 'constexpr'
        constexpr static int SIZE = 1024 * 256; //262 KB
```

Is there a way to force the compiler to use C++11 using meson?

--- a/README.md
+++ b/README.md
@@ -2,12 +2,20 @@
 A DS emulator that seeks to combine accuracy and speed. As of v0.1, it's a long way from either one, but with your support, CorgiDS will accomplish its goals.
 
 ## Compiling
-CorgiDS requires Qt 5.9 and currently supports qmake and meson. Please report any compilation errors or other problems that happen during this process.
+CorgiDS requires Qt 5.9 and currently supports qmake and meson. Please report any compilation errors or other problems that happen during this process. Pull requests are welcome for CMake support.
 
-## Building with meson
+### Building with qmake
+```
+cd CorgiDS/CorgiDS
+qmake CorgiDS.pro
+make
+```
+
+### Building with meson
 
 ```
-meson build && cd build
+meson build --buildtype=release
+cd build
 ninja
 ```
 

--- a/meson.build
+++ b/meson.build
@@ -1,4 +1,4 @@
-project('corgids', 'cpp')
+project('corgids', 'cpp', default_options : ['cpp_std=c++11'])
 qt5 = import('qt5')
 qt5dep = dependency('qt5', modules: ['Core','Gui', 'Widgets'])
 

